lib/scripts/game.js

```diff
@@ -53,29 +53,20 @@ let renderAllShapes = function(self, objectIndex) {
   }
 };
 
+let shapes = {
+  spoke: Spoke,
+  line: Line,
+  arc: Arc,
+  doubleArc: DoubleArc,
+  quadArc: QuadArc
+};
+
 let createTileObjects = function(self, levelArrangement) {
   for(let i=0; i<levelArrangement.length; i++) {
     let location = levelArrangement[i][0];
     let orientation = levelArrangement[i][2] || "all";
     let shapeType = levelArrangement[i][1];
-
-    switch (shapeType) {
-      case "spoke":
-        self.objectIndex[location] = (new Spoke(location, orientation));
-        break;
-      case "arc":
-        self.objectIndex[location] = (new Arc(location, orientation));
-        break;
-      case "line":
-        self.objectIndex[location] = (new Line(location, orientation));
-        break;
-      case "doubleArc":
-        self.objectIndex[location] = (new DoubleArc(location, orientation));
-        break;
-      case "quadArc":
-        self.objectIndex[location] = (new QuadArc(location, orientation));
-        break;
-    }
+    self.objectIndex[location] = new shapes[shapeType](location, orientation);
```

Comment:
Super cool refactor, I'm all about stuffing classes into objects.  I think I'd prefer if the object were named 'shape' or even 'shapeOfType' rather than 'shapes' so you can say 'new shape[shapeType]...' or 'new shapeOfType[shapeType]...'.